mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/MailboxElasticsearchConstants.java

```diff
@@ -19,10 +19,12 @@
 
 package org.apache.james.mailbox.elasticsearch;
 
+import org.apache.james.backends.es.AliasName;
 import org.apache.james.backends.es.IndexName;
 import org.apache.james.backends.es.TypeName;
 
 public interface MailboxElasticsearchConstants {
+    AliasName DEFAULT_MAILBOX_ALIAS = new AliasName("mailboxAlias");
```

Comment:
I would name that one mailbox and I would call the real index depending on its actual content (v1, v2, or anything that would help identifying its content)

Reply:
When you have a read and a write alias it makes sense IMO to have them called **mailboxWriteAlias** and **mailboxReadAlias**.

Regarding the index, I will add a version on it, if it please you.